### PR TITLE
Typo in CalWORKs exempt parameter

### LINKED ISSUE #5114: Typo in CalWORKs exempt parameter
https://github.com/PolicyEngine/policyengine-us/blob/5d1853053b1cc26c7a92abcc19c898ae58d61bc9/policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/exempt.yaml#L1

Should be "assistance unit".

But for consistency with other parameters, I'd suggest instead:
>California provides CalWORKs assistance units the Exempt Maximum Aid Payment if they participate in any of these program.

### PR DESCRIPTION
**Pull request by srilareddy**
Fixes #5114

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/exempt.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-description: An assistance receives the Exempt MAP in California CalWORKs when it receives one of the following benefits.
+description: California provides CalWORKs assistance units the Exempt Maximum Aid Payment if they participate in any of these program.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/exempt.yaml:1]:
```suggestion
description: California provides CalWORKs assistance units the exempt maximum aid payment if they participate in any of these programs.
```

File: policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/exempt.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/exempt.yaml:1]: add changelog

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+   changes:
+    fixed: 
+      - Corrected Typo in CalWORKs exempt parameter.
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]:
```suggestion
      - Corrected typo in CalWORKs exempt parameter.
```

**Comment by MaxGhenis** [comment]
probably needs a `git pull upstream master`

**Pull request merged**